data/templates/ocserv/ocserv_passwd.tmpl

```diff
@@ -0,0 +1,6 @@
+#<username>:<group>:<hash>
+{% for user in username %}
+{% if not "disable" in username[user] %}
```

Comment:
Please indent Jinja commands by two when nesting, there should be two additional whitespaces before "if not"